slidez-core/src/main/java/com/binarystudio/academy/slidez/infrastructure/ws/HelloMessage.java

```diff
@@ -0,0 +1,21 @@
+package com.binarystudio.academy.slidez.infrastructure.ws;
+
+public class HelloMessage {
+
+    private String name;
+
+    public HelloMessage() {
+    }
+
+    public HelloMessage(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
```

Comment:
Have a look at other models(maybe still in PRs), there are annotations @AllArgsConstructor and so on